packages/react/src/CTABanner/CTABanner.module.css

```
@@ -0,0 +1,82 @@
+.CTABanner {
+  position: relative;
+  max-width: 1280px;
+  margin: 0 auto;
+  display: flex;
+}
+
+.CTABanner--Shadow {
```

is there a reason for Shadow being uppercased here?

```suggestion
.CTABanner--shadow{
```

Alternatively, how about naming it after the prop to make it easier to correlate the prop with the class.

```suggestion
.CTABanner--hasShadow {
```

---

no, I will remove in next push